provider/marathon/marathon_test.go

```diff
@@ -934,440 +699,563 @@ func TestMarathonGetProtocol(t *testing.T) {
 		c := c
 		t.Run(c.desc, func(t *testing.T) {
 			t.Parallel()
+			provider := &Provider{}
 			actual := provider.getProtocol(c.application)
 			if actual != c.expected {
-				t.Errorf("got protocol '%s', want '%s'", actual, c.expected)
+				t.Errorf("actual %q, expected %q", actual, c.expected)
 			}
 		})
 	}
 }
 
-func TestMarathonGetPassHostHeader(t *testing.T) {
-	provider := &Provider{}
-
-	applications := []struct {
+func TestMarathonGetSticky(t *testing.T) {
+	cases := []struct {
+		desc        string
 		application marathon.Application
 		expected    string
 	}{
 		{
-			application: marathon.Application{
-				Labels: &map[string]string{}},
+			desc:        "label missing",
+			application: application(),
```

Comment:
`create` prefix is not fluent syntax 😉 (builders must be fluent, I think)
@vdemeester do that like me http://vincent.demeester.fr/posts/2017-01-01-go-testing-functionnal-builders/

But in this case, I prefer you don't change anything instead use a `create` prefix.